src/api/eventTypes.js

```diff
@@ -173,3 +175,236 @@ export type RestartEvent = $ReadOnly<{|
+  // realm_create_public_stream_policy
+  create_stream_policy: $PropertyType<InitialDataRealm, 'realm_create_stream_policy'>,
+  // realm_create_web_public_stream_policy
+
+  default_code_block_language: $PropertyType<InitialDataRealm, 'realm_default_code_block_language'>,
+
+  // realm_default_external_accounts doesn't have an event because it can
+  //   only change when the server is restarted anyway; see
+  //   https://chat.zulip.org/#narrow/stream/378-api-design/topic/.60value.60.20in.20realm.20op.3A.20update/near/1276269
+
+  // realm_default_language
+
+  // TODO(server-5.0): See notes in InitialDataRealm.
+  // realm_default_twenty_four_hour_time
+  // realm_delete_own_message_policy
+
+  description: $PropertyType<InitialDataRealm, 'realm_description'>,
+  digest_emails_enabled: $PropertyType<InitialDataRealm, 'realm_digest_emails_enabled'>,
+  digest_weekday: $PropertyType<InitialDataRealm, 'realm_digest_weekday'>,
+
+  // TODO: Number or boolean? Event doc and register-response doc disagree
+  disallow_disposable_email_addresses: empty,
+
+  // TODO(server-5.0): See note in InitialDataRealm.
+  edit_topic_policy: $PropertyType<InitialDataRealm, 'realm_edit_topic_policy'>,
+
+  email_address_visibility: $PropertyType<InitialDataRealm, 'realm_email_address_visibility'>,
+  email_auth_enabled: $PropertyType<InitialDataRealm, 'realm_email_auth_enabled'>,
+  email_changes_disabled: $PropertyType<InitialDataRealm, 'realm_email_changes_disabled'>,
+  emails_restricted_to_domains: $PropertyType<
+    InitialDataRealm,
+    'realm_emails_restricted_to_domains',
+  >,
+
+  // TODO(server-4.0): See note in InitialDataRealm.
+  realm_giphy_rating: $PropertyType<InitialDataRealm, 'realm_giphy_rating'>,
+
+  // TODO: Do servers send this, here or in InitialDataRealm?
+  // realm_google_hangouts_domain
+
+  icon_source: $PropertyType<InitialDataRealm, 'realm_icon_source'>,
+  icon_url: $PropertyType<InitialDataRealm, 'realm_icon_url'>,
+  inline_image_preview: $PropertyType<InitialDataRealm, 'realm_inline_image_preview'>,
+  inline_url_embed_preview: $PropertyType<InitialDataRealm, 'realm_inline_url_embed_preview'>,
+
+  // TODO(server-4.0): See note in InitialDataRealm.
+  // realm_invite_by_admins_only
+
+  invite_required: $PropertyType<InitialDataRealm, 'realm_invite_required'>,
+
+  // TODO(server-4.0): See note in InitialDataRealm.
+  // realm_invite_to_realm_policy
+
+  invite_to_stream_policy: $PropertyType<InitialDataRealm, 'realm_invite_to_stream_policy'>,
+  is_zephyr_mirror_realm: $PropertyType<InitialDataRealm, 'realm_is_zephyr_mirror_realm'>,
+  logo_source: $PropertyType<InitialDataRealm, 'realm_logo_source'>,
+  logo_url: $PropertyType<InitialDataRealm, 'realm_logo_url'>,
+  mandatory_topics: $PropertyType<InitialDataRealm, 'realm_mandatory_topics'>,
+  message_content_allowed_in_email_notifications: $PropertyType<
+    InitialDataRealm,
+    'realm_message_content_allowed_in_email_notifications',
+  >,
```

Comment:
I think this type is also a good place to do a `// prettier-ignore` so that it's possible to put the information in a more tabular form: all formatted the same way, and with the right-hand property names all starting at the same column so it's possible to scan down them easily by eye.

For example:
```js
  invite_to_stream_policy:
    $PropertyType<InitialDataRealm, 'realm_invite_to_stream_policy'>,
  is_zephyr_mirror_realm:
    $PropertyType<InitialDataRealm, 'realm_is_zephyr_mirror_realm'>,
  logo_source:
    $PropertyType<InitialDataRealm, 'realm_logo_source'>,
  logo_url:
    $PropertyType<InitialDataRealm, 'realm_logo_url'>,
  mandatory_topics:
    $PropertyType<InitialDataRealm, 'realm_mandatory_topics'>,
  message_content_allowed_in_email_notifications:
    $PropertyType<InitialDataRealm, 'realm_message_content_allowed_in_email_notifications'>,
```

(It's possible to edit from here to there pretty efficiently using multi-cursors: e.g. select `: $`, then hit Ctrl+D a bunch of times, then make the edit.)

Alternatively things could be kept on one line each, but that'd get awfully long in a few cases like `message_content_allowed_in_email_notifications`.